### DISCUSSION TITLE
Welcome to beancount Discussions!

### DISCUSSION THREAD

**Discussion by blais** [beancount/beancount#761] [General]
Start your discussion.

**Comment by gmkramer** [comment]
I am needing some advice on an error message I am getting when I try to a data convert/import from a .csv . When I perform a 'bean' function like bean-identify  (file.beancount), bean-web (file.beancount) or bean-extract  etc..I get the following error traceback:

 Traceback (most recent call last):
  File "/usr/bin/bean-extract", line 33, in <module>
    sys.exit(load_entry_point('beancount==2.3.3', 'console_scripts', 'bean-extract')())
  File "/usr/bin/bean-extract", line 25, in importlib_load_entry_point
    return next(matches).load()
StopIteration

I am getting it on versions 2.3 and 3.0. Any suggestions on what the problem this is referring to? 
--Much appreciated
